### ISSUE TITLE
Avatar: [A11y]In not focusable and pressable case, the avatar should not be hidden from screen readers

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#3012]
**Describe the bug**
When  the avatar is not focusable and pressable (**interactive = false**), it should be hidden from screen readers.

**Isolated Example**
UI5-WC-React Demo
https://tr4jt.csb.app/

UI5 Demo
https://sapui5.hana.ondemand.com/#/entity/sap.m.Avatar/sample/sap.m.sample.Avatar

**To Reproduce**
1. Turn on screen reader and open **UI5-WC-React Demo**
2. Click the avatar.  The screen reader will read 'Avatar'
3. Open **UI5 Demo** 
4. Click any non-focusable avatar, the screen will not read out any texts.

**Expected behavior**
When the avatar is not focusable and pressable then it should hide from screen readers.
Expect the same behavior like https://sapui5.hana.ondemand.com/#/entity/sap.m.Avatar/sample/sap.m.sample.Avatar

**Screenshots**
![image](https://user-images.githubusercontent.com/5724592/112438699-296a5a00-8d83-11eb-9bda-9106a04245a3.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: Current Version
`@ui5/webcomponents-react` version: Current Version
Operating System:  Windows
Browser: Chrome

**Additional context**
I am from SAP SuccessFactors

**Comment by Lukas742** [comment]
Hi @sleric1024 

Since this component is developed by the UI5 Web Components team, I have transferred this issue to their repo.

**Comment by kskondov** [comment]
Hello @SAP/ui5-webcomponents-topic-p could you please take a look if this is the expected behavior.
Thanks and Best Regards,
Konstantin

**Comment by dobrinyonkov** [comment]
Hi @sleric1024,

the non-interactive (interactive = false) avatar should still be accessible and available for the screen reader users. For example, when navigating in reading mode (Virtual PC cursor ON) avatar accessibility should still be read out to the users.
Moreover, testing the screen reader behavior with mouse/pointer inputs (clicks) is not an ordinary approach, as the common operable approach is to use a Keyboard Interface.

Kind Regards,
Dobrin
